### PR TITLE
Hw2

### PR DESCRIPTION
**Pull request by BadSolver**
1)Task: https://www.figma.com/file/16bdlWFoZXXIpyyZmwy9JU/Logoipsum
2)
image
3) deadline - 22.02.2022 / done 16.02.2022

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -13,10 +14,338 @@ h1,h2,h3,h4,h5,h6{
...
+    font-family: 'DM Sans', sans-serif;
+}
+.footer_card_btn2{
+    background: #9F3919;
+    border: none;
+    border-radius: 10px;
+    width: 305px;
+    height: 60px;
+    padding: 19px 34px 43px 37px;
+    color: white;
```
Comment (shuliakleonid) [inline style.css]: you should write color in one style 
if write color as HEX, write it

File: style.css [hunk 2]
```diff
@@ -13,10 +14,338 @@ h1,h2,h3,h4,h5,h6{
...
+    position: relative;
+    bottom: 20px;
+    
+}
+.name__footer_card2{
+    font-size: 18px;
+    line-height: 28px;
+    text-align: center;
+    letter-spacing: -0.562737px;
+    color: rgba(255, 255, 255, 0.7);
```
Comment (shuliakleonid) [inline style.css:307]: you should write color in one style
if write color as HEX, write it

File: style.css [hunk 3]
```diff
@@ -13,10 +14,338 @@ h1,h2,h3,h4,h5,h6{
...
+}
+.footer_name{
+    width: 523px;
+    font-size: 60px;
+    line-height: 70px;
+    letter-spacing: -1.87579px;
+    color: #19191B;
+}
+.first_foot_bnt{
+    color: white;
```
Comment (shuliakleonid) [inline style.css]: you should write color in one style
if write color as HEX, write it

File: style.css [hunk 4]
```diff
@@ -13,10 +14,338 @@ h1,h2,h3,h4,h5,h6{
...
+}
+.hero_btn{
+    margin-top: 50px;
+    
+    font-family:'DM Sans', sans-serif;
+    font-size: 17px;
+    line-height: 22px;
+    text-align: center;
+    letter-spacing: -0.531474px;
+    color: white;
```
Comment (shuliakleonid) [inline style.css]: you should write color in one style
if write color as HEX, write it

File: style.css [hunk 5]
```diff
@@ -13,10 +14,338 @@ h1,h2,h3,h4,h5,h6{
...
+    margin-left: 159px;
+    margin-right: 180px;
+}
+.btn__inner{
+    font-family:'DM Sans', sans-serif;
+    font-size: 17px;
+    line-height: 22px;
+    text-align: center;
+    background-color: #FF7143;
+    color:white ;
```
Comment (shuliakleonid) [inline style.css]: you should write color in one style
if write color as HEX, write it

File: style.css [hunk 6]
```diff
@@ -13,10 +14,338 @@ h1,h2,h3,h4,h5,h6{
...
+    border-radius: 10px;
+    padding: 19px 60px 19px 60px;
+    border: none;
+    letter-spacing: -0.5px;
+    width: 211px;
```
Comment (shuliakleonid) [inline style.css]: cursor: pointer should be on all button

File: style.css [hunk 7]
```diff
@@ -13,10 +14,338 @@ h1,h2,h3,h4,h5,h6{
...
+    font-family:'Mulish', sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 30px;
+    letter-spacing: -0.625263px;
+    color: #696871;
+}
+.hero_btn{
+    margin-top: 50px;
```
Comment (shuliakleonid) [inline style.css:96]: cursor: pointer should be on all button

File: style.css [hunk 8]
```diff
@@ -13,10 +14,338 @@ h1,h2,h3,h4,h5,h6{
...
+    font-family:'DM Sans', sans-serif;
+    font-size: 0.9375rem;
+    line-height: 19px;
+}
+.nav__link:hover{
+    color:#19191B ;
+    text-decoration:underline;
+}
+.nav{
+    padding-top: 44px;
```
Comment (shuliakleonid) [inline style.css]: not good practice move block with padding
you should move a block with margin, padding exist for move inner content

File: style.css [hunk 9]
```diff
@@ -13,10 +14,338 @@ h1,h2,h3,h4,h5,h6{
...
     margin-left: 184px;
+    padding-right: 123px;
 }
-.wrapper{
+.header__inner{
     display: flex;
+    justify-content: space-around;
+}
+.nav__link{
+    padding-top: 44px;
```
Comment (shuliakleonid) [inline style.css:35]: cursor: pointer

File: style.css [hunk 10]
```diff
@@ -1,9 +1,10 @@
-font-family: 'DM Sans', sans-serif;
-font-family: 'Mulish', sans-serif;
+/*font-family: 'DM Sans', sans-serif;
```
Comment (shuliakleonid) [inline style.css:1]: try write code with this guide https://codeguide.academy/html-css.html#css

File: index.html [hunk 11]
```diff
@@ -3,28 +3,142 @@
...
-                            <li>Blog</li>
-                        </ul>
-                    </nav>
+    <header class="header">
+        <div class="container">
+            <div class="header__inner">
+                <div class="logo_header">
+                    <img src="/TMS/image/logoheader.jpg" alt="logo__header">
                 </div>
+                <nav class="nav">
```
Comment (shuliakleonid) [inline index.html:20]: the named classes are not quite correct, but this is not a problem, we will discuss this in the next lesson

**Review by shuliakleonid — CHANGES_REQUESTED** [review]

File: index.html (hunk 11 above)
Comment (BadSolver) [reply index.html:20]: ok

File: style.css (hunk 10 above)
Comment (BadSolver) [reply style.css:1]: ok

File: style.css (hunk 9 above)
Comment (BadSolver) [reply style.css:35]: done

File: style.css (hunk 1 above)
Comment (BadSolver) [reply style.css]: done

File: style.css (hunk 3 above)
Comment (BadSolver) [reply style.css]: done

File: style.css (hunk 5 above)
Comment (BadSolver) [reply style.css]: done

File: style.css (hunk 7 above)
Comment (BadSolver) [reply style.css:96]: done

File: style.css (hunk 8 above)
Comment (BadSolver) [reply style.css]: done

File: style.css (hunk 4 above)
Comment (BadSolver) [reply style.css]: done

File: style.css (hunk 2 above)
Comment (BadSolver) [reply style.css:307]: Not ready.
How to set transparency in HEX format?

Comment (shuliakleonid) [reply style.css:307]: transparency  don't exist in HEX format

**Review by shuliakleonid — APPROVED** [review]

File: index.html [hunk 12]
```diff
@@ -3,28 +3,142 @@
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Logoipsum</title>
     <link rel="stylesheet" href="style.css">
     <link rel="preconnect" href="https://fonts.googleapis.com">
```
Comment (shuliakleonid) [inline index.html:7]: you should write './style.css'

**Review by shuliakleonid — CHANGES_REQUESTED** [review]

**Review by shuliakleonid — APPROVED** [review]

**Review by egkirpichev — APPROVED** [review]
Well done!

**Review by AlexisKozlov — APPROVED** [review]
Great!

**Pull request merged**